File: MyMechanicAgenda/app/src/main/java/com/santosdaniel/mymechanicagenda/helper/StringHelper.kt
package com.santosdaniel.mymechanicagenda.helper

/**
 * Container methods to the manage strings
 */
object StringHelper {
    const val EMPTY_STRING = ""
    const val SPACE = " "

    /**
     * Checks if a string is null or empty
     */
    fun isNullOrEmpty(str: String?) = (str == null) || (str.isEmpty())

    /**
     * Check that is string is neither null or empty
     */
    fun isNotNullOrEmpty(str: String?) = (!isNullOrEmpty(str))

}

File: MyMechanicAgendaIOS/MyMechanicAgendaIOS/helper/StringHelper.swift
import Foundation

/// Helper to handle strings
public class StringHelper {
    public static let emptyString = ""
    public static let space = " "

    /**
     * Checks if a string is null or empty
     */
    public class func isNullOrEmpty(_ str: String?) -> Bool {
        if let string = str {
            return string.isEmpty
        } else {
            return true
        }
    }

    /**
     * Check that is string is neither null or empty
     */
    public class func isNotNullOrEmpty(_ str: String?) -> Bool {
        return (!isNullOrEmpty(str))
    }
}
